charmcraft/templates/init-go-framework/src/charm.py.j2

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# Copyright {{ year }} {{ author }}
+# See LICENSE file for licensing details.
+
+"""Go Charm entrypoint."""
+
+import logging
+import typing
+
+import ops
+
+import paas_app_charmer.go
+
+logger = logging.getLogger(__name__)
+
+
+class GoCharm(paas_app_charmer.go.Charm):
```

Comment:
I know this is the same in the Django and Flask extensions, but we might want to go back to using the same as our other templates:

```suggestion
class {{ GoCHarm }}(paas_app_charmer.go.Charm):
```

Reply:
I have updated the three templates to use this code. Thanks!